airflow/models.py

```diff
@@ -201,10 +202,11 @@ def get_dag(self, dag_id):
                     dag.last_loaded < orm_dag.last_expired
                 )
         ):
-            # Reprocessing source file
+            # Then reprocess source file
```

Comment:
Ah it's kind of hard to understand since the comment is split up, I still prefer the older version (you don't need to read the comment above to understand what this comment does).

It's a nit so feel free to ignore.